CONTRIBUTING.md

```diff
@@ -36,7 +34,52 @@ To set up an OpenHTF dev environment, follow the steps below.
 1. `sudo apt-get install python-pip swig libssl-dev python-dev libffi-dev`
 2. `sudo pip install --upgrade pip`
 3. `sudo pip install virtualenv`
-4. `(cd to openhtf directory)`
+4. (cd to openhtf directory)
```

Comment:
While you're poking around here, I would actually make this a code block instead of each line separately, and do `# cd to openhtf directory`, making it much more copy-paste friendly.  as is it's kinda tedious.